DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -69,5 +75,18 @@ private LandingPageViewModel GetLandingPageViewModel(int sectionIndex)
                 CurrentSiteBaseUrl = configuration.GetCurrentSystemBaseUrl(),
             };
         }
+
+        private LearningContentLandingPageViewModel GetLearningContentLandingPageViewModel(
+            IEnumerable<LearningContentSummary> learningContents
+        )
+        {
+            return new LearningContentLandingPageViewModel
+            {
+                MiniHubNavigationModel = new MiniHubNavigationModel(LandingPageMiniHubName, sections, 2),
+                UserIsLoggedIn = User.Identity.IsAuthenticated,
+                CurrentSiteBaseUrl = configuration.GetCurrentSystemBaseUrl(),
+                LearningContents = learningContents,
+            };
+        }
     }
 }

```

Comment:
I think we can get rid of this private method and create a constructor in the view model for this